### DISCUSSION TITLE
PhotoPrism Quits after a few minutes

### DISCUSSION THREAD

**Discussion by droidus** [photoprism/photoprism#3130] [Q&A]
I deployed a new instance of PhotoPrism with the docker run command.  After a few minutes of running fine and being able to upload a few photos, it unexpectedly quits:
` docker ps -a
CONTAINER ID   IMAGE                   COMMAND                  CREATED          STATUS                       PORTS     NAMES
8c7369cb9b3f   photoprism/photoprism   "/scripts/entrypoint…"   17 minutes ago   Exited (143) 7 minutes ago             photoprism
`
docker container logs  doesn't seem to reveal anything.  I am running Ubuntu 22.04 as a LXC container in Proxmox.

**Comment by graciousgrey** [comment]
Have you seen https://docs.photoprism.app/getting-started/docker/#troubleshooting already ?

**Comment by droidus** [comment]
Went to try and start the docker container to analyze ram usage, and got this message:
`Error response from daemon: failed to create endpoint photoprism on network bridge: failed to add the host (veth50b6b71) <=> sandbox (veth75e632a) pair interfaces: cannot allocate memory
Error: failed to start containers: 8c7369cb9b3f
`

**Comment by droidus** [comment]
It seems I am able to start it now.  but it still quits unexpectedly.  It doesn't seem to be running out of memory.  I don't see it getting max'd out.
